confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -131,7 +131,7 @@ public Map<String, DataPartitionTable> allocateDataPartition(
   @Override
   public void run() {
     int balanceCount = 0;
-    while (true) {
+    while (!configManager.isStopped()) {
```

Comment:
You can rollback changes in LoadManager, because we fix this problem in [IOTDB-3522](https://github.com/apache/iotdb/pull/6328)

Reply:
> You can rollback changes in LoadManager, because we fix this problem in [IOTDB-3522](https://github.com/apache/iotdb/pull/6328)

ok,revert.